sqlalchemy_ibmi/base.py

```diff
@@ -674,17 +677,15 @@ class DB2Dialect(default.DefaultDialect):
     preparer = DB2IdentifierPreparer
     execution_ctx_cls = DB2ExecutionContext
 
-    _reflector_cls = ibm_reflection.DB2Reflector
-
     def __init__(self, **kw):
-        super(DB2Dialect, self).__init__(**kw)
+        super(IBMiDb2Dialect, self).__init__(**kw)
```

Comment:
```suggestion
        super().__init__(**kw)
```
In Python 3 we don't need to use the class name. We may need to deal with the use of multiple inheritance, though: https://www.datacamp.com/community/tutorials/super-multiple-inheritance-diamond-problem